fenzo-core/src/main/java/com/netflix/fenzo/queues/tiered/ResAllocsUtil.java

```diff
@@ -0,0 +1,92 @@
+package com.netflix.fenzo.queues.tiered;
+
+import com.netflix.fenzo.queues.QueuableTask;
+import com.netflix.fenzo.sla.ResAllocs;
+import com.netflix.fenzo.sla.ResAllocsBuilder;
+
+import java.util.List;
+
+/**
+ * Collection of helper functions for {@link com.netflix.fenzo.sla.ResAllocs} data type.
+ */
+public final class ResAllocsUtil {
+
+    public static ResAllocs addAll(List<ResAllocs> resToAdd) {
+        if (resToAdd.isEmpty()) {
+            throw new IllegalArgumentException("Empty list provided");
```

Comment:
I am throwing an exception here, as without a single ResAllocs instance, the taskGroupName is unknown. Having said that, I agree that exception is not a good choice here. I will change this to Optional.